### DISCUSSION TITLE
Sliderendcircle fadein

### DISCUSSION THREAD

**Discussion by SitrisB** [ppy/osu#23217] [Ideas] [answered]
Issue: Sliderendcircle fades in at a much slower rate than sliderstartcircle/hitcircle. 

Some things I've already tried that don't resolve the issue:
adding sliderendcircle and sliderstartcircle files to the skin
toggling hidden on/off
toggling snaking in&out sliders on/off

Proposed solution:
There should be a toggle for this behaviour in lazer settings/skin.ini

**Comment by peppy** [comment]
Please provide a video of what you are seeing, and please attach your skin.

**Comment by SitrisB** [comment]
Hello Peppy, thanks for your reply. I had some issues with recording software, but here you are: 

Using the default (2014) skin produces the same behaviour. 

120fps playbackspeed 10%. left: lazer, right: stable
https://www.youtube.com/shorts/8xZ0JnzLb4k

**Reply by peppy** [reply]
Thanks a lot for going to the effort of making this comparison. I've determined that the animation was incorrect, but only when snaking is disabled. Will follow up with a fix.

**Reply by peppy** [reply, marked as answer]
#23372
